<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>limitDialog</class>
 <widget class="QDialog" name="limitDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>93</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Searc limit</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelLimit">
     <property name="text">
      <string>Set search limit (0 = no limit):</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="limitLayout">
     <item>
      <widget class="QLineEdit" name="lineEditLimit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <property name="rightMargin">
      <number>100</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
